web/client/epics/widgetsbuilder.js

```diff
@@ -56,17 +56,32 @@ export const initEditorOnNew = (action$, {getState = () => {}} = {}) => action$.
     }, {step: 0})));
 export const initEditorOnNewChart = (action$, {getState = () => {}} = {}) => action$.ofType(NEW_CHART)
     .filter(() => widgetBuilderAvailable(getState()))
-    .switchMap((w) => Rx.Observable.of(closeFeatureGrid(), editNewWidget({
-        legend: false,
-        mapSync: true,
-        cartesian: true,
-        yAxis: true,
-        widgetType: "chart",
-        filter: wfsFilter(getState()),
-        ...w,
-        // override action's type
-        type: undefined
```

Comment:
no, because the action does not contain parameters so it does make sense to spread an action with only type just to turn type to undefined

![image](https://github.com/geosolutions-it/MapStore2/assets/19175505/0fc8b75c-344a-49c8-9b72-180760e3c80c)

Reply:
ok